packages/core/src/components/ui/Notifications/_Notifications.scss

```diff
@@ -64,6 +64,7 @@ $notification-font-color: $light-gray5;
   // LEVELS
   &__level {
     margin-bottom: $ufx-base-size * 0.6;
+    position: relative;
```

Comment:
could you explain why this style is needed ? just wanted to know reason to make sure it doesn't affect unintended issues in hf-ui and across ufx/example app

Reply:
because all close buttons are positioned on top of first message. close button has absolute position, but its parent has static, so all close buttons are positioned relative to the notifications container 
![Снимок экрана от 2022-01-10 12-59-18](https://user-images.githubusercontent.com/81973123/148755261-2c295412-b898-45d0-affa-aaf8a274a98f.png)